transport/grpc/handler.go

```diff
@@ -74,9 +75,9 @@ func (h *handler) handle(srv interface{}, serverStream grpc.ServerStream) (err e
 	}
 	switch handlerSpec.Type() {
 	case transport.Unary:
-		return h.handleUnary(ctx, transportRequest, serverStream, streamMethod, start, handlerSpec.Unary())
+		return h.handleUnary(ctx, transportRequest, serverStream, streamMethod, start, middleware.ApplyUnaryInbound(handlerSpec.Unary(), h.i.t.options.unaryInboundInterceptor))
```

Comment:
Could you plz put each argument on a new line (here and at the line 80). Want to make a line shorter, but more important, to make the call to middleware.ApplyUnaryInbound as obvious as it possible.